cg/clients/chanjo2/api.py

```diff
@@ -0,0 +1,40 @@
+"""Coverage analysis API for human clinical sequencing data."""
```

Comment:
Consider renaming this file to `client.py` or `chanjo_api_client.py` or something